node/pkg/watchers/near/watcher.go

```diff
@@ -171,7 +173,10 @@ func (e *Watcher) runChunkFetcher(ctx context.Context) error {
 				continue
 			}
 			for _, job := range newJobs {
-				e.schedule(ctx, job, job.delay)
+				err := e.schedule(ctx, job, job.delay)
+				if err != nil {
+					logger.Warn("near.schedule failed", zap.String("error", err.Error()))
```

Comment:
```suggestion
					logger.Warn("near.schedule failed", zap.Error(err))
```

See [zap.Error()](https://pkg.go.dev/go.uber.org/zap#Error) godoc.